packages/framework/src/Resources/scripts/admin/orderItems.js

```diff
@@ -96,8 +96,8 @@
                 var index = $orderItem.data('index');
 
                 $collection.append($orderItem);
-                $($('#order_form_items')).jsFormValidator('addPrototype', index);
 
+                Shopsys.validation.addNewItemToCollection('#js-order-items', index);
```

Comment:
This replacement looks very unintuitive, can you please explain? I would expect:

```js
$($('#js-order-items')).jsFormValidator('addPrototype', index);

Reply:
done